photon-core/src/main/java/org/photonvision/vision/camera/USBCameraSource.java

```diff
@@ -212,21 +280,23 @@ public HashMap<Integer, VideoMode> getAllVideoModes() {
 
                         videoModesList.add(videoMode);
 
+                        // TODO - do we want to trim down FPS modes? in cases where the camera has no gain control,
+                        // lower FPS might be needed to ensure total exposure is acceptable.
                         // We look for modes with the same height/width/pixelformat as this mode
                         // and remove all the ones that are slower. This is sorted low to high.
                         // So we remove the last element (the fastest FPS) from the duplicate list,
                         // and remove all remaining elements from the final list
-                        var duplicateModes =
-                                videoModesList.stream()
-                                        .filter(
-                                                it ->
-                                                        it.height == videoMode.height
-                                                                && it.width == videoMode.width
-                                                                && it.pixelFormat == videoMode.pixelFormat)
-                                        .sorted(Comparator.comparingDouble(it -> it.fps))
-                                        .collect(Collectors.toList());
-                        duplicateModes.remove(duplicateModes.size() - 1);
-                        videoModesList.removeAll(duplicateModes);
+                        // var duplicateModes =
+                        //         videoModesList.stream()
+                        //                 .filter(
+                        //                         it ->
+                        //                                 it.height == videoMode.height
+                        //                                         && it.width == videoMode.width
+                        //                                         && it.pixelFormat == videoMode.pixelFormat)
+                        //                 .sorted(Comparator.comparingDouble(it -> it.fps))
+                        //                 .collect(Collectors.toList());
+                        // duplicateModes.remove(duplicateModes.size() - 1);
```

Comment:
Why's this logic removed?

Reply:
copying from discord but - this was removed to allow lower framerates to be selected to support broader exposure ranges, which are useful for apriltag